hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/XceiverServerRatis.java

```diff
@@ -516,6 +515,15 @@ public RaftServer getServer() {
     return server;
   }
 
+  public RaftServer.Division getServerDivision() throws IOException {
```

Comment:
This part of a change is contained in master (https://github.com/apache/ozone/pull/1728)

Reply:
I see. so this is from the cherry picked commit from master.